docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -14,305 +14,194 @@ weight: 700
+
+| Setting                      | Required | Description                                                                                                                                                                                    | Default         |
+| ---------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- |
+| `enabled`                    | No       | Whether generic OAuth2 authentication is allowed.                                                                                                                                              | `false`         |
+| `name`                       | No       | Name used to refer to the generic OAuth2 authentication from Grafana's user interface.                                                                                                         | `OAuth`         |
+| `icon`                       | No       | Icon used for the generic OAuth2 authentication in Grafana's user interface.                                                                                                                   | `signin`        |
+| `client_id`                  | Yes      | Client ID provided by your OAuth2 app.                                                                                                                                                         |                 |
+| `client_secret`              | Yes      | Client secret provided by your OAuth2 app.                                                                                                                                                     |                 |
+| `auth_url`                   | Yes      | Authorization endpoint of your OAuth2 provider.                                                                                                                                                |                 |
+| `token_url`                  | Yes      | Endpoint used to obtain OAuth2 access token.                                                                                                                                                   |                 |
+| `api_url`                    | Yes      | Endpoint used to obtain user information compatible with [OpenID UserInfo](https://connect2id.com/products/server/docs/api/userinfo).                                                          |                 |
+| `auth_style`                 | No       | Name of the [OAuth2 AuthStyle](https://pkg.go.dev/golang.org/x/oauth2#AuthStyle) to be used when ID token is requested from OAuth2 provider.                                                   | `AutoDetect`    |
```

Comment:
```suggestion
| `auth_style`                 | No       | Name of the [OAuth2 AuthStyle](https://pkg.go.dev/golang.org/x/oauth2#AuthStyle) to be used when ID token is requested from the OAuth2 provider.                                                   | `AutoDetect`    |
```

Reply:
I would add a link from line 75 to the auth_style section you have below. Are you not able to fit in the contents from that section into this table? If it's too much information, add a link like "For more information about `auth_style`, refer to [auth_style](link)."